src/coreComponents/mesh/generators/VTKUtilities.cpp

```diff
@@ -1435,17 +1435,17 @@ std::vector< localIndex > getWedgeNodeOrderingFromPolyhedron( vtkCell * const ce
   {
     auto edgeNode0 = cell->GetEdge( iEdge )->GetPointId( 0 );
     auto edgeNode1 = cell->GetEdge( iEdge )->GetPointId( 1 );
-    auto it0 = std::find( &nodeTri0[0], &nodeTri0[3], edgeNode0 );
-    auto it1 = std::find( &nodeTri0[0], &nodeTri0[3], edgeNode1 );
+    auto it0 = std::find( nodeTri0.begin(), nodeTri0.end(), edgeNode0 );
```

Comment:
I do prefer this version but if we are sure that we are dealing with triangles I do not understand how it's different from the original one.

Reply:
Because bound checks for std::vector::[], the original one breaks when the test is compiled in Debug mode.